From 1182a75e88af8250a64869bd5ea607061fac6339 Mon Sep 17 00:00:00 2001
From: Zhou Xuemei <zhouxuemei@wxiat.com>
Date: Tue, 7 Jun 2022 17:42:15 +0800
Subject: [PATCH] sw64: dts: rename spi flash partition to fix warning
Patch-mainline: NO, OTHERS
References: openEuler-22.03-LTS
Modified-by-SEL: No


Sunway inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

The two spi flash partitions in chip3.dts are both called "test" which
will raise a warning:

> sysfs: cannot create duplicate filename /bus/nvmem/devices/test

Rename them to spares0 and spares1 to fix it.

Signed-off-by: Zhou Xuemei <zhouxuemei@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/boot/dts/chip3.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/sw_64/boot/dts/chip3.dts b/arch/sw_64/boot/dts/chip3.dts
index ce61dfe6e7bd..29d1fd153509 100644
--- a/arch/sw_64/boot/dts/chip3.dts
+++ b/arch/sw_64/boot/dts/chip3.dts
@@ -131,7 +131,7 @@ partitions {
 					#size-cells = <1>;
 
 					partition@0 {
-						label = "test";
+						label = "spares0";
 						reg = <0 0x400000>;
 					};
 				};
@@ -153,7 +153,7 @@ partitions {
 					#size-cells = <1>;
 
 					partition@0 {
-						label = "test";
+						label = "spares1";
 						reg = <0 0x400000>;
 					};
 				};
-- 
2.34.1

